README.md

```diff
@@ -27,91 +19,24 @@ There are several scripts and helpers in this repository a release
-```
-$ release-notify <release version>
-```
-
-### Official release
-
-**Stage it**
-```
-$ gcbmgr stage release-1.10 --official
-```
-**Release it**
-(The complete invocation below is output at the end of the staging log)
-```
-$ gcbmgr release release-1.10 --buildversion=<staged version>
-```
-
-**Announce it**
-(The complete invocation below is output at the end of the release log)
-```
-$ release-notify <release version>
-```
-
-### Release with --nomock
-
-The above example workflows run *mock* versions of the release.  To produce
-a fully deployed and announced release, add `--nomock` to the command line.
-
-
-## All Tools
-
-* [gcbmgr](https://github.com/kubernetes/release/blob/master/gcbmgr) : GCB manager - find status and initiate builds
-* [anago](https://github.com/kubernetes/release/blob/master/anago) : Release Tool
-* [branchff](https://github.com/kubernetes/release/blob/master/branchff) : Fast-forward branching helper
-* [find_green_build](https://github.com/kubernetes/release/blob/master/find_green_build) : Ask Jenkins for a good build to use
-* [release-notes](https://github.com/kubernetes/release/blob/master/cmd/release-notes) : Scrape github for release notes \(See below for more info\)
-* [prin](https://github.com/kubernetes/release/blob/master/prin) : What tags/releases is my PR IN?
-* [changelog-update](https://github.com/kubernetes/release/blob/master/changelog-update) : Update CHANGELOG.md version entries by rescanning github for text and label changes
-* [push-build.sh](https://github.com/kubernetes/release/blob/master/push-build.sh) : Push a developer (or CI) build up to GCS
-* [script-template](https://github.com/kubernetes/release/blob/master/script-template) : Generate a script template in the kubernetes/release ecosystem
+Tools | Description
+ :---: | --
+[`gcbmgr`](https://github.com/kubernetes/release/blob/master/gcbmgr) | Google Cloud Builder Manager: <br/><br/> This is the main entry point for release managers for producing releases. All release types can be staged and later released using this method.
+[`anago`](https://github.com/kubernetes/release/blob/master/anago) | Release Tool: <br/><br/> The main driver for creating staged builds and releases. This is what runs inside GCB after a job is submitted using `gcbmgr`.
+[`branchff`](https://github.com/kubernetes/release/blob/master/branchff) | Fast-forward branching helper : <br/><br/> A tool used to pull new patches onto the release branch.
+<br/> [`find_green_build`](https://github.com/kubernetes/release/blob/master/find_green_build) <br/><br/> | Asks Jenkins for a good build to use.
+<br/> [`release-notes`](https://github.com/kubernetes/release/blob/master/cmd/release-notes) <br/><br/> | Scrape GitHub for release notes. See [Release Notes Gathering](#release-notes-gathering) for more information.
+<br/> [`prin`](https://github.com/kubernetes/release/blob/master/prin) <br/><br/> | What tags/releases is my PR IN?
```

Comment:
This was verbosely copied over from the old README. Does this simply mean, "_To show tags/releases that a PR is in_."  ?

Reply:
Yes.

E.g. when I am in the directory of the k8s source and call the script with a random PR number or a random commit hash, it shows which tags those PR/commit is included in. However, I have never ever used that script before.

```
user@b3a1b221edab:/usr/local/google/branchff-release-1.12/src/k8s.io/kubernetes$ /src/prin '69500'
Skipping security_layer::hosted_map extensions...

COMMITS: 3f102c3980 (initial)  (cherrypicks)
(https://github.com/kubernetes/kubernetes/pull/69500)

                LATENCY TO APPEAR (DAYS)
TAG             K8S
---             ---
v1.13.0         49.22
v1.13.0-alpha.1 3.10
v1.13.0-alpha.2 11.62
v1.13.0-alpha.3 18.57
v1.13.0-beta.0  26.01
v1.13.0-beta.1  35.42
v1.13.0-beta.2  41.26
v1.13.0-rc.1    46.32
v1.13.0-rc.2    47.36
v1.13.1         60.78
v1.13.1-beta.0  52.10
v1.13.2         88.56
v1.13.2-beta.0  61.67
v1.13.3         111.97
v1.13.3-beta.0  90.21
v1.13.4         137.53
v1.13.4-beta.0  112.06
v1.13.5-beta.0  138.79
v1.14.0-alpha.0 25.94
v1.14.0-alpha.1 94.68
v1.14.0-alpha.2 108.62
v1.14.0-alpha.3 122.58
v1.14.0-beta.0  129.81
v1.14.0-beta.1  136.64
v1.15.0-alpha.0 129.77

user@b3a1b221edab:/usr/local/google/branchff-release-1.12/src/k8s.io/kubernetes$ /src/prin '3f102c3980'
Skipping security_layer::hosted_map extensions...

COMMITS: 3f102c3980 (initial)  (cherrypicks)

                LATENCY TO APPEAR (DAYS)
TAG             K8S
---             ---
v1.13.0         49.22
v1.13.0-alpha.1 3.10
v1.13.0-alpha.2 11.62
v1.13.0-alpha.3 18.57
v1.13.0-beta.0  26.01
v1.13.0-beta.1  35.42
v1.13.0-beta.2  41.26
v1.13.0-rc.1    46.32
v1.13.0-rc.2    47.36
v1.13.1         60.78
v1.13.1-beta.0  52.10
v1.13.2         88.56
v1.13.2-beta.0  61.67
v1.13.3         111.97
v1.13.3-beta.0  90.21
v1.13.4         137.53
v1.13.4-beta.0  112.06
v1.13.5-beta.0  138.79
v1.14.0-alpha.0 25.94
v1.14.0-alpha.1 94.68
v1.14.0-alpha.2 108.62
v1.14.0-alpha.3 122.58
v1.14.0-beta.0  129.81
v1.14.0-beta.1  136.64
v1.15.0-alpha.0 129.77

user@b3a1b221edab:/usr/local/google/branchff-release-1.12/src/k8s.io/kubernetes$

```